puppet/Rakefile

```diff
@@ -11,7 +11,6 @@ PuppetSyntax.exclude_paths = ["vendor/**/*"]
 require 'puppet-lint/tasks/puppet-lint'
 PuppetLint::RakeTask.new :lint do |config|
   config.pattern = PuppetSyntax.manifests_paths
-  config.ignore_paths = ['modules/exim/**/*.pp']
```

Comment:
Rebased and updated this line to fully get rid of it. Waiting for CI now to see if it's still happy.